sdk/src/main/scala/io/horizen/account/state/ScAddrOwnershipLinkedList.scala

```diff
@@ -0,0 +1,89 @@
+package io.horizen.account.state
+
+import io.horizen.account.state.MessageProcessorUtil.NativeSmartContractLinkedList
+import io.horizen.account.state.NativeSmartContractMsgProcessor.NULL_HEX_STRING_32
+import io.horizen.account.utils.WellKnownAddresses.MC_ADDR_OWNERSHIP_SMART_CONTRACT_ADDRESS
+import io.horizen.evm.Address
+import sparkz.crypto.hash.Blake2b256
+
+import java.nio.charset.StandardCharsets
+
+class ScAddrOwnershipLinkedList(scAddressStringNoPrefix : String)
+  extends NativeSmartContractLinkedList {
+
+  val listTipKey: Array[Byte] = Blake2b256.hash(scAddressStringNoPrefix+"TipKey")
+  val listTipNullValue: Array[Byte] = Blake2b256.hash(scAddressStringNoPrefix+"TipNullValue")
+
+  val scAddressTipValue : Array[Byte] = Blake2b256.hash(scAddressStringNoPrefix)
+
+  private def getScAddrData(view: BaseAccountStateView, dataKey: Array[Byte]): Option[String] = {
+    val data = view.getAccountStorageBytes(MC_ADDR_OWNERSHIP_SMART_CONTRACT_ADDRESS, dataKey)
+    if (data.length == 0) {
+      // getting a not existing key from state DB using RAW strategy
+      // gives an array of 32 bytes filled with 0, while using CHUNK strategy, as the api is doing here
+      // gives an empty array instead
+      None
+    } else {
+      Some(new String(data, StandardCharsets.UTF_8))
+    }
+  }
+
+  def getItem(view: BaseAccountStateView, nodeRef: Array[Byte]): (String, Array[Byte]) = {
```

Comment:
I'm wondering if this `getItem` method could be in common with the `getScAddresRefsListItem` of `ScAddressRefsLinkedList.scala` class; they really looks similar except for the returning pair's first element

Reply:
In principle any linked list could use whatever return type it is necessary, here for instance we have a ByteSerializable obj versus a String